CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 x.x.x Release notes (yyyy-MM-dd)
 ==============================================================
 
+**This project is in the Beta stage. All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
+
+### Enhancements
+* Support anonymous application credentials ([#443](https://github.com/realm/realm-dart/pull/443/))
+
+x.x.x Release notes (yyyy-MM-dd)
```

Comment:
I tried to describe a new Beta version on top of the currently developed. That's way I changed the text to: `This project is in the **Beta** stage`. But actually we don't have much to release before Sync. So that I will combine ` value from write transaction callbacks` and the sync in one version.

Reply:
I added `Support anonymous application credentials` under `Features`. Is that correct?